### ISSUE TITLE
Sound import

### ISSUE THREAD

**Issue by kueda** [inaturalist/iNaturalistReactNative#2448]
Stub to add the ability to import a sound from the filesystem. We support this functionality in the legacy Android app and it's useful for folks who use other sounds recording / editing apps.

Immediately design need is how the user enters this flow. Another button in the advanced add obs wheel? Something in the sounds recorder?

**Comment by kvangork** [comment]
iOS does have methods for file browsing, but we should consider simplifying the design needs by making this something we support only through external initiation (#2449), through the app's share extension/intent which currently handles photos.

**Comment by abhasinat** [comment]
I agree, let's make sharing sounds into the app a priority and add importing down the line if necessary.

**Comment by kvangork** [comment]
This issue is worth keeping open  too. Users may want to add an audio recording as additional evidence to an existing observation, which would not be possible via the primary method of sharing into the app.
